### PR TITLE
Move PP() and PP2() macros to separate header file

### PR DESCRIPTION
**Pull request by Rogier-5**
Instead of redefining them everywhere...

### CODE REVIEW

**Comment by sofar** [comment]
:+1:

File: src/util/macros.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (sfan5) [inline src/util/macros.h]: extra newline

also maybe these can be moved to `basic_macros.h`

**Comment by Rogier-5** [comment]
Rebased.

**Comment by Zeno-** [comment]
:+1:

**Comment by paramat** [comment]
4d4b8bb8a46b6472d86fa848954dbc26b4fadb50

**Pull request closed without merging**